ci/scenarios/vPool/add_remove_alba_vpool_test/main.py

```diff
@@ -153,7 +153,8 @@ def validate_add_extend_remove_vpool(timeout=ADD_EXTEND_REMOVE_VPOOL_TIMEOUT):
                 start = time.time()
                 try:
                     AddRemoveVPool._add_vpool(vpool_name=AddRemoveVPool.VPOOL_NAME, fragment_cache_cfg=cfg, api=api,
-                                              albabackend_name=hdd_backend.name, timeout=timeout, preset_name=AddRemoveVPool.PRESET['name'],
+                                              block_cache_cfg=cfg, albabackend_name=hdd_backend.name, timeout=timeout,
+                                              preset_name=AddRemoveVPool.PRESET['name'],
                                               storagerouter_ip=storagerouter_ip)
                 except TimeOutError:
```

Comment:
Predetermine if its OSE or EE.
installed_versions = PackageManager.get_installed_versions(client=compute_client)
 is_ee = 'volumedriver-ee-base' in installed_versions
        if is_ee is True: